.github/workflows/Test Thunder.yml

```diff
@@ -95,6 +95,7 @@ jobs:
         cmake -G Ninja -S Thunder -B ${{matrix.build_type}}/build/Thunder \
         -DBINDING="127.0.0.1" \
         -DCMAKE_BUILD_TYPE=${{matrix.build_type}} \
+        -DBUILD_SHARED_LIBS=ON \
```

Comment:
Again, just to make sure - do we want to run the test only with shared libs enabled from now on?